functions.opengraph.php

```diff
@@ -100,10 +104,12 @@ function jetpack_og_tags() {
 
 		$tags['og:url']         = get_permalink( $data->ID );
 		if ( ! post_password_required() ) {
-			if ( ! empty( $data->post_excerpt ) ) {
-				$tags['og:description'] = preg_replace( '@https?://[\S]+@', '', strip_shortcodes( wp_kses( $data->post_excerpt, array() ) ) );
+			$excerpt = get_the_excerpt( $data );
```

Comment:
thanks for pointing me to that bug and fix!

sadly, the workaround of filtering on `jetpack_open_graph_base_tags` in the markdown plugin wouldn't help in this case, since `og:description` has URLs stripped before that, which breaks the original markdown link syntax.